src/library/interface.cpp

```diff
@@ -83,4 +94,24 @@ void OrbSlam2Interface::convertOrbSlamPoseToKindr(const cv::Mat& T_cv,
   *T_kindr = Transformation(q_kindr, t_kindr);
 }
 
+void OrbSlam2Interface::getBodyTransform() {
+  cv::FileStorage fsSettings(settings_file_path_, cv::FileStorage::READ);
```

Comment:
Please load settings using rosparams. I can show you how to do this if you like.